Starting out

ztmurphy21 opened the pull request:
Here is what I have started please go off the latest commit, not the first one...

Commit:
left side nav built (sorta)

I think we should make icons for these and put those in rather than text
so it will work better on mobile.

Commit:
icons loaded on navigation

ztmurphy21 commented:
Changes I've done today for the group.

Commit:
iframe is cool!!!!!

Commit:
all pages have their perspective forum

iframe linked all pages to their perspective forum, need to decide home
page and the design and color scheme of the app.

Commit:
can't forget electrical and robotic majors...

Commit:
lots of changes

made adjustments so that it works on screen sizes mobile. so far have
tested on iPhone 6/6S and iphone 6/6S plus with expected results.

Commit:
works and fits on all mobile screen sizes!!!!!

after four hours of messing with it.

Commit:
one more change

ztmurphy21 commented:
#2

froschs commented:
I forget how to clone/download files from a pull request... I am just going to copy and paste the code from here then to take a look at it.

froschs commented:
The iframe section of the app was too small when I was testing it. I removed the iframe section of the CSS files on mine and changed the iframe call to this: 
`<iframe  src="URL" frameborder="0" style="overflow: hidden; height: 100%;
        width: 75%; position: absolute;" height="100%" width="75%"></iframe>`
It worked better for me but try it and let me know what you think.

ztmurphy21 commented:
Did you upload it to FileZilla?

---

From: froschs notifications@github.com
Sent: Tuesday, September 13, 2016 2:10:56 PM
To: ztmurphy21/Web-Content-Development--Mobile-App-Project
Cc: Zachary Murphy; Author
Subject: Re: [ztmurphy21/Web-Content-Development--Mobile-App-Project] Starting out (#1)

The iframe section of the app was too small when I was testing it. I removed the iframe section of the CSS files on mine and changed the iframe call to this:

<iframe src="URL" frameborder="0" style="overflow: hidden; height: 100%;
width: 75%; position: absolute;" height="100%" width="75%"></iframe>

It worked better for me but try it and let me know what you think.

## 

You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHubhttps://github.com/ztmurphy21/Web-Content-Development--Mobile-App-Project/pull/1#issuecomment-246771603, or mute the threadhttps://github.com/notifications/unsubscribe-auth/ANPz50j29MFq19pMFvWI6ulv6ryEAitqks5qpucwgaJpZM4JzU0D.

Commit:
Seth F. made changes

Commit:
assingment upload

Commit:
added Samuels icons

Samuel did a great job designing the icons, going to have to make some
tweaks but looking even better.

Commit:
colors!!!!

Commit:
transfer page fixed

broke transfer page, but i fixed it. Seth F fixed ProBoards text size
issue.

Commit:
moved icons and working on listOfMajors.html

listOfMajors.html still needs some touch up.

Commit:
revamed

Seth F's idea.

Commit:
update-sync
